test/test-fs.c

```diff
@@ -1069,6 +1099,39 @@ TEST_IMPL(fs_mkdtemp) {
 }
 
 
+TEST_IMPL(fs_mkstemp) {
+  int r;
+  const char* path_template = "test_file_XXXXXX";
+
+  loop = uv_default_loop();
+
+  r = uv_fs_mkstemp(loop, &mkstemp_req1, path_template, mkstemp_cb);
+  /* There's no cross-OS way to check an fd is valid */
```

Comment:
this function is called the async way, so it doesn't return the fd, thus this comment is misleading.